src/com/rcv/Main.java

```diff
@@ -23,75 +23,89 @@
-  // purpose: create config object from command line args
-  // param: args command line arguments passed into main program
+  // purpose: create config object
+  // param: path to config file
   // returns: the new ElectionConfig object or null if there was a problem
-  public static ElectionConfig makeElectionConfig(String [] args) {
+  static ElectionConfig makeElectionConfig(String configPath) {
     // config: the new object
     ElectionConfig config = null;
-    if (args.length > 0) {
+    try {
       // rawConfig holds the basic election config data parsed from json
       RawElectionConfig rawConfig;
       // parse raw config
-      rawConfig = JsonParser.parseObjectFromFile(args[0], RawElectionConfig.class);
+      rawConfig = JsonParser.parseObjectFromFile(configPath, RawElectionConfig.class);
       // create and validate new ElectionConfig
       if (rawConfig != null) {
         config = new ElectionConfig(rawConfig);
+        // set up log output
+        Logger.setup(config.auditOutput());
+        Logger.log("Parsed config file: %s", configPath);
+        Logger.log("Logging to: %s", config.auditOutput());
         if (!config.validate()) {
           Logger.log("There was a problem validating the election configuration.");
           Logger.log("Please see the README.txt for details.");
+          config = null;
         }
       }
-    } else {
-      System.err.println("No election configuration file specified as input.");
-      System.err.println("Please see the README.txt for details.");
+    } catch (IOException exception) {
+      System.err.print(
+          String.format(
+              "Failed to configure logging output to file: %s\n%s",
+              config.auditOutput(),
+              exception.toString()
+          )
+      );
     }
+    // TODO: Should probably add either check for null config here (and throw exception?), or whenever it's called
     return config;
   }
 
+  // function: executeTabulation
+  // purpose: execute tabulation for given ElectionConfig
+  // param: config object containing cvr file paths to parse
+  static String executeTabulation(ElectionConfig config) {
+    // Read cast vote records from cvr files
+    // castVoteRecords will contain all cast vote records parsed by the reader
+    List<CastVoteRecord> castVoteRecords;
+    String response = "Tabulation successful!";
+    try {
+      // parse the cast vote records
+      castVoteRecords = parseCastVoteRecords(config);
+      // tabulator for tabulation logic
+      Tabulator tabulator = new Tabulator(castVoteRecords, config);
+      // do the tabulation
+      tabulator.tabulate();
+      // generate visualizer spreadsheet data
+      tabulator.generateSummarySpreadsheet();
+      // generate audit data
+      tabulator.doAudit(castVoteRecords);
+    } catch (Exception exception) {
+      response = String.format("ERROR during tabulation: %s", exception.toString());
+      Logger.log(response);
+    }
+    return response;
+  }
```

Comment:
Cool, looks good.  It does seem a little odd to return a user-facing status string, but it's fine for now.